frontend/src/Utilities/Date/formatDateTime.js

```diff
@@ -10,15 +11,15 @@ function getRelativeDay(date, includeRelativeDate) {
   }
 
   if (isYesterday(date)) {
-    return 'Yesterday, ';
+    return `${translate('Yesterday')}, `;
```

Comment:
This is going to be another place where we need to have more context to translate properly. In this case, I think we just want to return the relative day without the comma then translate it again in `formatDateTime`, which could look something like: 

```
const relativeDay = getRelativeDay(date, includeRelativeDay);
const formattedDate = moment(date).format(dateFormat);
const formattedTime = formatTime(date, timeFormat, { includeMinuteZero: true, includeSeconds });

if (relativeDay) {
  return translate('{relativeDay}, ${formattedDate} ${formattedTime}', { relativeDay, formattedDate, formattedTime });
}

return translate('${formattedDate} ${formattedTime}', { formattedDate, formattedTime });
```

This way different locales that prefer the relative day at the end or time before the date can be handled better.